### ISSUE TITLE
update bloc library

### ISSUE THREAD

**Issue by linxydmhg** [adityadroid/Messio#51]
version 1 removes dispatch(), currentState

**Comment by adityadroid** [comment]
> version 1 removes dispatch(), currentState

Yes. I read about the new release. I'll update the code. Thanks!
